WordPress/Classes/Models/Blog.h

```diff
@@ -192,4 +193,16 @@ typedef NS_ENUM(NSInteger, SiteVisibility) {
  */
 - (WordPressComApi *)restApi;
 
+/**
+ Returns a REST API client if available
+
+ If the blog is a WordPress.com one or it has Jetpack it will return a REST API
+ client. Otherwise, the XML-RPC API should be used.
+
+ @warning this method doesn't know if a Jetpack blog has the JSON API disabled
+
+ @return a WordPressComRestApi object if available
+ */
+- (WordPressComRestApi *)comRestApi;
```

Comment:
For consistency with `WPAccount`, I'd call this `wordPressComRestApi`. Also, make sure to mark it `_Nullabe` once #5325 is merged.